src/EnergyPlus/PackagedTerminalHeatPump.cc

```diff
@@ -5046,6 +5066,25 @@ namespace PackagedTerminalHeatPump {
 		PTUnit( PTUnitNum ).LatHeatEnergy = PTUnit( PTUnitNum ).LatHeatEnergyRate * ReportingConstant;
 		PTUnit( PTUnitNum ).ElecConsumption = PTUnit( PTUnitNum ).ElecPower * ReportingConstant;
 
+		if ( PTUnit( PTUnitNum ).FirstPass ) { // reset sizing flags so other zone equipment can size normally
+
+			if( !SysSizingCalc ) {
+
+				if ( CurZoneEqNum > 0 ) {
```

Comment:
@rraustad Not suggesting this be done in this PR, but it seems like it would be good to have a utility function like resetSizingData that everybody calls to reset these flags.  Maybe as part of the fix for #5728?